ruby/cross_service_examples/item-tracker/spec/db_wrapper_spec.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
```

Comment:
Apache license.

Reply:
In Ruby we throw the license in LICENSE.txt and attach to gem. In sample code, is it really necessary?